content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -20,72 +20,61 @@ This page describes the RuntimeClass resource and runtime selection mechanism.
 
 ## Runtime Class
 
-RuntimeClass is an alpha feature for selecting the container runtime configuration to use to run a
-pod's containers.
+RuntimeClass is a beta feature (as of v1.14) for selecting the container runtime configuration to
+use to run a pod's containers.
 
-### Set Up
-
-As an early alpha feature, there are some additional setup steps that must be taken in order to use
-the RuntimeClass feature:
-
-1. Enable the RuntimeClass feature gate (on apiservers & kubelets, requires version 1.12+)
-2. Install the RuntimeClass CRD
-3. Configure the CRI implementation on nodes (runtime dependent)
-4. Create the corresponding RuntimeClass resources
+{{< warning >}} RuntimeClass includes *breaking* changes in the beta upgrade in v1.14. If you were
+using RuntimeClass prior to v1.14, the following actions are required:
```

Comment:
I think you should make this a separate section with a new header "Upgrading RuntimeClass from Alpha to Beta"

Reply:
Done